plugins/org.eclipse.xtext/src/org/eclipse/xtext/formatting2/regionaccess/internal/NodeEObjectRegion.java

```diff
@@ -47,4 +48,11 @@ public AbstractElement getGrammarElement() {
 		return null;
 	}
 
+	@Override
+	public String getText() {
+		if (node instanceof RootNode)
```

Comment:
Please don't use implementation classes. This looks like you want to do node.getRootNode().getText()